controller/internal/routes/edge_router_router.go

```diff
@@ -47,11 +47,11 @@ func (ir *EdgeRouterRouter) Register(ae *env.AppEnv) {
 	sr := registerCrudRouter(ae, ae.RootRouter, ir.BasePath, ir, permissions.IsAdmin())
 	registerCrudRouter(ae, ae.RootRouter, ir.BasePathLegacy, ir, permissions.IsAdmin())
 
-	servicesUrl := fmt.Sprintf("/{%s}/%s", response.IdPropertyName, EntityNameService)
-	servicesListHandler := ae.WrapHandler(ir.ListServices, permissions.IsAdmin())
+	serviceEdgeRouterPoliciesUrl := fmt.Sprintf("/{%s}/%s", response.IdPropertyName, EntityNameServiceEdgeRouterPolicy)
+	serviceEdgeRouterPoliciesListHandler := ae.WrapHandler(ir.ListServiceEdgeRouterPolicies, permissions.IsAdmin())
 
-	sr.HandleFunc(servicesUrl, servicesListHandler).Methods(http.MethodGet)
```

Comment:
Providing an endpoint of effective services is still possible. This change makes it so that we expose the internal model of ER->SERPs and that is fine.

We will want those "effective" endpoints in the future for security auditing.

Reply:
Agree. I know we're going to have to do another pass where we add a bunch more endpoints for UI awesomeness.